internal/server/proto/server.proto

```diff
@@ -296,6 +296,18 @@ service Waypoint {
   // JSON formatting is not supported.
   rpc WaypointHclFmt(WaypointHclFmtRequest) returns (WaypointHclFmtResponse);
 
+  //----------------------------------------------------------------------
+  // Ondemand runner functions.
+  //----------------------------------------------------------------------
+  rpc UpsertOndemandRunner(UpsertOndemandRunnerRequest) returns (UpsertOndemandRunnerResponse);
```

Comment:
I think this should be `UpsertOndemandRunnerConfig` (the `Config`) part.

Reply:
`UpsertOnDemandRunnerConfig` per our slack convo 😄